docs/source/create_channel/create_channel_participation.md

```diff
@@ -4,19 +4,21 @@ Fabric v2.3 introduces the ability to create channels between network members wi
+ * **System channel no longer required**: Previously a channel managed by the ordering service (called the "system channel") was defined using the `configtxgen` tool to create a genesis block. This channel was exclusively managed by the ordering service and was the template on which all application channels were based. Besides the creation of the system representing an extra step (as compared to the new process), this system channel created an extra layer of administration that, for some use cases, provided no tangible benefit.
+ * **Consortium no longer required**: You no longer need to define the set of organizations, known as the "consortium", who are permitted to create channels on a particular ordering service. With this new process, **all channels are application channels**, so the concept of a list of organizations who can create channels no longer applies. Any set of organizations can get together and create a channel using a defined set of ordering nodes (which become the ordering service of that channel). Additionally, if the MSP of the peer organization, as listed in the system channel, needs to be changed, the peer organization to coordinate with an orderer organization to change the MSP.
+ * **Simplified ordering node creation process**: Because a genesis block no longer needs to exist before an ordering node is created, admins can now focus on the process of setting up their infrastructure and making sure their node is working correctly before joining a particular application channel. Additionally, because an ordering node no longer needs to scan the system channel to determine which channels it is joined to and then replicate those ledgers, the node starts up more rapidly.
 
 **Benefits of the new process:**
 
- * **Increased Privacy** - A single orderer does not know about all the channels on the ordering service.
- * **Scalability** - When there is a large number of ordering service nodes (orderers) on the system channel, starting the orderer can take a long time as it has to wait for each orderer to replicate the channel ledger.
- * **Operational benefits** - Updating an application channel no longer requires write permissions from a separate ordering organization. Also, you now have a way to list all the channels an orderer is a member of. Lastly, you have the ability to easily remove a channel and its respective resources from an orderer.
+ * **Increased privacy**: Because all ordering nodes used to be joined to the system channel, every ordering node in a network knew about the existence of every channel on that ordering service, even if the node itself wasn't a consenter on that channel and therefore wasn't ordering or store its blocks. Now, an ordering node only knows about the channels it is joined to.
```

Comment:
`therefore wasn't ordering or storing its blocks` (not sure why the suggestion option isn't appearing here)

Reply:
Done